parsl/tests/test_staging/test_staging_stdout.py

```diff
@@ -15,6 +15,7 @@ def output_to_stds(*, stdout=parsl.AUTO_LOGNAME, stderr=parsl.AUTO_LOGNAME):
     return "echo hello ; echo goodbye >&2"
 
 
+@pytest.mark.shared_fs
```

Comment:
this is a test of staging, not of the shared file system: the stdout file should be moved by the staging code, not across a shared filesystem